common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -792,13 +802,31 @@ div.video {
   &.is-touch {
     div.tc-wrapper {
       article.video-wrapper {
-        object, iframe, video{
+        object, iframe, video {
           width: 100%;
           height: 100%;
         }
       }
     }
   }
+
+  .poster-youtube {
```

Comment:
NIT: I would consider using a base .video-pre-roll class, and add &.html5 as a special case nested within that base class.